packages/ibm-products/src/components/UserAvatar/UserAvatar.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Import portions of React that are needed.
+import React from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg /*, carbon */ } from '../../settings';
+
+import { User } from '@carbon/react/icons';
+
+import { Button } from '@carbon/react';
+// Carbon and package components we use.
+/* TODO: @import(s) of carbon components and other package components. */
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--user-avatar`;
+const componentName = 'UserAvatar';
+
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+// Default values can be included here and then assigned to the prop params,
+// e.g. prop = defaults.prop,
+// This gathers default values together neatly and ensures non-primitive
+// values are initialized early to avoid react making unnecessary re-renders.
+// Note that default values are not required for props that are 'required',
+// nor for props where the component can apply undefined values reasonably.
+// Default values should be provided when the component needs to make a choice
+// or assumption when a prop is not supplied.
+
+// Default values for props
+// const defaults = {
+//   /* TODO: add defaults for relevant props if needed */
+// };
+
+/**
+ * TODO: A description of the component.
+ */
+
+const defaults = {
+  renderIcon: () => <User size={32} />,
+  tooltipAlignment: 'bottom',
+  tooltipText: 'Iam tooltip',
+};
+
+export let UserAvatar = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+      backgroundColor,
+      className,
+      /* TODO: add other props for UserAvatar, with default values if needed */
+      renderIcon = defaults.renderIcon,
+      theme,
```

Comment:
I think for this part of the review, we should avoid using a `theme` prop. The next part of the UserAvatar epic will look into if we can accomplish this without a `theme` prop. In the original user profile image, this causes some unexpected behavior since the theme is controlled both by the actual Carbon theme used and by a separate theme prop.

Reply:
Removed in the next PR so marking as okay for now.